docs/Configuration.md

```diff
@@ -129,22 +131,22 @@ or
 * `DllPlugin`:
   * create Dynamicaly Linked Library with vendored modules
   * no automatic rebuilds
-  * prefered for code that change rerely
+  * prefered for code that changes rarely
   * shared between parallel builds
   * great for bigger projects
 
-`CommonsChunkPlugin` is the default used plugin, since it doesn't require any changes to be made or commands
+`CommonsChunkPlugin` is used by default, since it doesn't require any changes to be made or commands
 to be run.
 
 > Please note, that `CommonsChunkPlugin` and `DllPlugin` are __not interoperable__, so it's up to you to decide
 which one suits your project better.
 
 If you want to use `DllPlugin`, you firstly need to execute `gluestick build --vendor` (for production add `NODE_ENV=production` when running the command), before building any other app.
-This command will create files (including your vendor DLL bundle) under `build/assets/dlls`. Now if you `build` or `start` an app or whole project, it will use vendor DLL bundle.
+This command will create files (including your vendor DLL bundle) under `build/assets/dlls`. Now if you `build` or `start` an app or the whole project, it will use the vendor DLL bundle.
 
-`DllPlugin` is the essential part that allows for app builds parallelisation.
-In order to split app builds you, firstly need to have vendor DLL bundle compiled and available for each
-build thread/process, then to build a single app use `gluestick build --client -A /<appName>` command.
+`DllPlugin` is the essential part that allows for app build parallelisation.
+In order to split app builds you, firstly you need to have the vendor DLL bundle compiled and available for each
```

Comment:
Change to:

"In order to split app builds, you'll first need to have the vendor DLL[...]"